platform/mellanox/mlnx-onie-fw-update.sh

```diff
@@ -106,7 +115,7 @@ system_reboot() {
     sleep 5s
 
     # Use SONiC reboot scenario
-    /usr/local/bin/reboot
+    /usr/local/bin/reboot -f
```

Comment:
@DavidZagury why do we need force here? Did u try without force?

Reply:
@nazariig Yes, I tried without the force, this is what solved the issue we discussed.